### DISCUSSION TITLE
View past seasonal backgrounds

### DISCUSSION THREAD

**Discussion by Phippe** [ppy/osu#13092] [Ideas]
**Describe the new feature:** At the moment you can view each seasonal background only until the next one arrives. There have been so many good past seasonal backgrounds, it's almost a shame that we can't view them anymore.

**Proposal designs of the feature:** Include an option in the menu to select year and season of a background to display. Maybe restrict it to the ones of the current season, but from past years. Or make it a random seasonal background each time you start the game.
This feature could be supporter only, I'm not sure about that one though.

**Comment by mcendu** [comment]
Needs #7055 to be resolved first
